README.md

```diff
@@ -0,0 +1,41 @@
+[ ![js-standard-style](https://img.shields.io/badge/code%20style-standard-brightgreen.svg)](http://standardjs.com)
+
+# Shift Next
+## Purpose
+
+This repo holds everything related to next js, this is meant to be used with the reference site. Pages are set up with algolia and redux. This repo also holds all express route handlers for calls made by shift-node-api to the shift platform. All you need to do is change some .env variables to work with the shift platform and you will have a working front-end.
```

Comment:
We need to think about writing these as if it's another developer from one of our clients, these docs are too contextual to how we'd perceive it

Reply:
I agree with @cbarratt - we should explain that this library can be used for building a Shift-powered ecommerce front-end site with Next and how it provides a collection of pages and backend endpoints which can be customised.

It is going to be a public repo representing us and our product so I'd try and keep it a bit more formal as well. On the most basic level, avoid abbreviations and make sure we capitalise correctly: repo -> repository, next js -> Next.js, shift -> Shift or SHIFT, etc.